### PR TITLE
feature: initial bsc vault implementations

### PR DESCRIPTION
**Pull request by axejintao**

### CODE REVIEW

File: src/config/chain.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+export class BinanceSmartChain implements Chain {
+	public name = 'BinanceSmartChain';
+	public chainId = '0x61';
+	public tokens = [];
+	public setts = [];
+	public provider = new ethers.providers.JsonRpcProvider(Provider.Binance);
+}
+
+export const eth = new Ethereum();
```
Comment (ethkaneki) [inline src/config/chain.ts:30]: Should these live within modules or services?

File: src/config/constants.ts [hunk 2]
```diff
@@ -42,4 +43,8 @@ export const MERKLE_CLAIM_BUCKET = process.env.MERKLE_CLAIM_BUCKET || 'MISSING R
 // third party api constants
 export const CURVE_API_URL = 'https://www.curve.fi/raw-stats/apys.json';
 export const COINGECKO_URL = 'https://api.coingecko.com/api/v3/simple';
-export const ETHERS_JSONRPC_PROVIDER = new ethers.providers.JsonRpcProvider('https://cloudflare-eth.com/');
+
+export enum Provider {
```
Comment (ethkaneki) [inline src/config/constants.ts:47]: I think this should be moved to a config service at some point. Doesn't need to be in this PR, but just writing it down.

**Review by ethkaneki — APPROVED** [review]
Minor comments, but can be addressed with future changes

File: src/config/chain.ts (hunk 1 above)
Comment (axejintao) [reply src/config/chain.ts:30]: this will be multi module, we can talk about what to do with this. i'll make an issue for these

**Pull request merged**